### PR TITLE
Test cleanup

### PR DESCRIPTION
**Pull request by bmwoolf**
- remove reentrancy
- add ReentrancyGuard locally for static analyzers
- remove unused vars

### CODE REVIEW

File: src/CypherEscrow.sol [hunk 1]
```diff
@@ -159,6 +157,7 @@ contract CypherEscrow is ReentrancyGuard {
         if (txInfo.origin == address(0)) revert NotValidAddress();
```
Comment (zkSoju) [inline src/CypherEscrow.sol:157]: We can remove this line because we perform the check in escrow

File: src/CypherEscrow.sol [hunk 2]
```diff
@@ -180,20 +177,22 @@ contract CypherEscrow is ReentrancyGuard {
...
+
         // Send ETH back
         if (txInfo.asset == address(0x0)) {
             (bool success, ) = address(to).call{value: txInfo.amount}("");
             if (!success) revert TransferFailed();
         } else {
             // Send ERC20 back
             /// TODO: this could be a potential exploit
-            address token = txInfo.asset;
+            address token_ = txInfo.asset;
```
Comment (zkSoju) [inline src/CypherEscrow.sol:190]: Is there a reason for this?

**Review by zkSoju — CHANGES_REQUESTED** [review]

File: src/CypherEscrow.sol (hunk 2 above)
Comment (bmwoolf) [reply src/CypherEscrow.sol:190]: it conflicts with a read from state `address public token;`

Comment (zkSoju) [reply src/CypherEscrow.sol:188]: lets name it `asset` then

**Pull request merged**